route_linux.go

```diff
@@ -1533,6 +1533,7 @@ type RouteGetOptions struct {
 	Iif      string
 	IifIndex int
 	Oif      string
+	OifIndex int
```

Comment:
I was wondering why not making this `uint32` then I see LifIndex is already there as `int`.

LGTM